beaker_kernel/lib/utils.py

```diff
@@ -164,18 +164,6 @@ def register_magic(fn):
     return register_magic
 
 
-def togglable_tool(env_var, *, name: str | None = None):
```

Comment:
If you remove this here, it will break existing contexts outside of this repo.
Any reason to not just leave it in as deprecated?

Reply:
https://github.com/search?q=repo%3ADARPA-ASKEM%2Fbeaker-kernel+togglable_tool&type=code

The places where it's used should just be deleted. It's duplicating functionality that already exists in beaker-kernel

Reply:
If [the answer](https://github.com/jataware/beaker-kernel/pull/47#discussion_r1594569798) to this other question below is yes, we're going to have to make modifications to askem-beaker anyway

Reply:
Perhaps, but you are making things harder on yourself by enforcing synchronicity on deployments that isn't needed.
If the other changes you made make this decorator not do anything then it's ok to remove the code and leave a warning.

```
import warnings
def togglable_tool(env_var, *, name: str | None = None):
    warnings.warn("This decorator is deprecated and will be removed in version 1.6.0")
    def noop(fn):
        return fn
    return noop
```